Editorial recommendation for Test Procedure for Audio Description (SC 1.2.5) (#422)
atbcb/ICTTestingBaseline: _baselines/17SyncMedia.md

</ol>

#### Test Results

<p id="4TR">If any of the above checks fail, then Baseline 17.4-CaptionsPrerecorded fails.</p>

### 17.5 Test Procedure for Audio Description (Prerecorded)

**Baseline Test ID:** 17.5-ADPrerecorded

#### Identify Content

<p id="5IC">Pre-recorded synchronized multimedia.  Do not include media that is clearly labeled as a <a href="https://www.w3.org/WAI/WCAG22/Understanding/audio-description-prerecorded#dfn-media-alternative-for-text">media alternative for text</a>.  (An example of media alternative for text is ASL version of content.)</p>

#### Test Instructions

<ol id="5TI">

>     <li id="5TI-1">Enable audio descriptions through multimedia player functions and play the media.  If a separate media file with audio descriptions is provided, test that file.  If there is not a separate medial file, and there is not an option to enable audio description, test the default file.</li>

Reviewer: ```suggestion
    <li id="5TI-1">Enable audio descriptions through multimedia player functions (if available) and play the media.  If a separate media file with audio descriptions is provided, test that file. </li>
```

Reviewer 2: Agreed

Reviewer 3: Did you mean to say "medial" file in line 128, or should it be "media" file?

Reviewer 2: media

>     <li id="5TI-2">Check that the narration (with audio descriptions enabled) adequately describes important visual content in the media, including information about actions, characters, scene changes, on-screen text, and other visual content. [SC 1.2.5]</li>

Reviewer: Audio includes sounds and speech. Narration is just speech.
 
- Would a train whistle be adequate to describe a train passing?

Reviewer 2: Agreed

Reviewer: @cliffbert, what do you agree with?
